app/frontend/components/pages/receiveAda/addressItem.tsx

```diff
@@ -1,125 +1,127 @@
-import {h, Component} from 'preact'
-import {connect} from '../../../helpers/connect'
-import actions from '../../../actions'
+import {Fragment, h} from 'preact'
 import {ADALITE_CONFIG} from '../../../config'
 import CopyOnClick from '../../common/copyOnClick'
+import AddressVerification from '../../common/addressVerification'
+import * as QRious from '../../../libs/qrious'
+
+type ViewOnProps = {
+  name: string
+  url: string
+  inline?: boolean
+}
+
+const ViewOn = ({name, url, inline}: ViewOnProps): h.JSX.Element =>
+  inline ? (
+    <a className="address-link" href={url} target="_blank" rel="noopener">
+      {name}
+    </a>
+  ) : (
+    <Fragment>
+      View on{' '}
+      <a className="address-link" href={url} target="_blank" rel="noopener">
+        {name}
+      </a>
+    </Fragment>
+  )
```

Comment:
Although small in size, I'd move it to a separate file. I see we use also "ViewOnCardanoScan", so it would perhaps make sense to merge it and use just that

Reply:
I moved it to separate file, but haven't merged it. I think it complicate things more then make it clearer.